astro-docs/src/content/docs/concepts/Module Federation/micro-frontend-architecture.mdoc

```diff
@@ -147,7 +146,7 @@ export default config;
 ```
 
 {% aside type="note" title="More details" %}
-You can return any configuration [object that webpack's Module Federation supports](https://webpack.js.org/plugins/module-federation-plugin/#sharing-hints).
+You can return any configuration [object that webpack/rspack's Module Federation supports](https://webpack.js.org/plugins/module-federation-plugin/#sharing-hints).
```

Comment:
nit/question: this links to webpack docs, but mentioned RsPack. is the api 1:1 or should we reference individually to webpack and rspack docs ?